gslb/nodes/avi_model_nodes.go

```diff
@@ -181,11 +185,17 @@ func (v *AviGSObjectGraph) GetHmChecksum() uint32 {
 
 func (v *AviGSObjectGraph) CalculateChecksum() {
 	// A sum of fields for this GS
-	var memberIPs []string
 	var memberObjs []string
+	var memberAddrs []string
 
 	for _, gsMember := range v.MemberObjs {
-		memberIPs = append(memberIPs, gsMember.IPAddr+"-"+strconv.Itoa(int(gsMember.Weight)))
+		var server string
+		if gsMember.VirtualServiceUUID != "" {
+			server = gsMember.VirtualServiceUUID + "-" + gsMember.ControllerUUID
```

Comment:
You seem to only check if the virtualserviceuuid is empty or not and not the same for controlleruuid - but the code blocks above seems to account for an empty controllerUUID as well, in such a case the server would look like: `vsuuid-` - is that a valid way of populating the server?

Reply:
Thanks, fixed. We shouldn't hit this condition anymore.